frontend/src/App.jsx

```diff
@@ -13,12 +13,25 @@ function App() {
   const { sessionDetails } = useSessionStore();
   const { alertDetails } = useAlertStore();
 
+  const btn = (
+    <Space>
```

Comment:
@jagadeeswaran-zipstack Do we need the `<Space>` component here? Since we only have `<Button>` inside it, I don't think it will be required. Please verify and make the necessary changes.